aiida/tools/dbimporters/plugins/icsd.py

```diff
@@ -604,6 +617,29 @@ def query_page(self):
             for i in self.soup.find_all('input', type='checkbox'):
                 self._results.append(i['id'])
 
+    def query_db_remarks(self, idnum):
+        """
+        Query the mysql db icsd_remarks table for the input crystal.
+        Explanations for all the remarks are included in the mysql "standard_remarks" table.
+        """
+        query_string = f'Select * from icsd_remarks WHERE IDNUM = {idnum}'
+        self.cursor.execute(query_string)
+        self.db.commit()
+        remarks = []
+        for row in self.cursor.fetchall():
+            remarks.append((row[1], row[2]))
```

Comment:
If you are going this way of individual queries for each CIF, shouldn't we at least not here make sure there is just one match and return just the one? Feel it is weird to return a list when only one result is expected and then having `query_db_is_theoretical` do `for remark in remarks: if remark[0]`

Reply:
It was just to have the function to potentially also parse the other remarks, but i agree that doing this prematurely probably didn't make sense.